container-host-files/etc/scf/config/scripts/patches/fix_haproxy_fd_requirements.sh

```diff
@@ -7,6 +7,10 @@ set -e
 # https://www.pivotaltracker.com/story/show/128789361
 # See HCF-1065 for the outcome of running performance tests on HCF.
 
+# It further contains a temporary patch needed to make kube-dns work
+# for BPM-managed jobs.  This will go away with the transition to the
+# cf-operator, which will not require BPM anymore.
```

Comment:
That starting comment block is wrong now (it's no longer patching a pre-start script); please reword the whole thing so it makes sense (and better describes why it needs to be in one patch file, rather than two separate patches).

Of course, it _would_ be preferable if this could be in two patch files instead (patching the same target file on disk), so that it's easier to remove one after we do the BPM thing

Reply:
Will check

Reply:
Done with commit d18bf53f7e914783360b615d9d38c2ffc3739cea